### PR TITLE
[15628] XML configuration for Ownership QoS

### PR DESCRIPTION
**Pull request by MRicoIE2C**
XML examples added to show the possibility of configuring ownership QoS policy from XML.
Related to Fast DDS PR: https://github.com/eProsima/Fast-DDS/pull/2967.

### CODE REVIEW

**Review by jsantiago-eProsima — CHANGES_REQUESTED** [review]
This comment is actually only tangentially related to this PR. Now that we are modifying things related to ownership, we should clarify Ownership's behavior when two writers have the same strength. This is something that slipped past my Ownership review.

**Comment by MRicoIE2C** [comment]
> This comment is actually only tangentially related to this PR. Now that we are modifying things related to ownership, we should clarify Ownership's behavior when two writers have the same strength. This is something that slipped past my Ownership review.

Done: 42e068e9 Refs #15628: Explain case of same strengths

File: docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst [hunk 1]
```diff
@@ -733,6 +733,8 @@ There are two possible values (see |OwnershipQosPolicyKind-api|):
   The owner can be changed dynamically according to the highest `strength` between the alive DataWriters, which has not
   violated the deadline contract concerning the data instances.
   That `strength` can be changed using the :ref:`ownershipstrengthqospolicy`.
+  In case two DataWriters have the same `strength` value,
+  the DataWriter with minimum `GUID` value would be the owner of the topic.
```
Comment (jsantiago-eProsima) [inline docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst:737]:
```suggestion
  the DataWriter with a lower `GUID` value would be the owner of the topic.
```
Small nitpick

Comment (MRicoIE2C) [reply docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst]: Done:
2ca357ce Refs #15628: Nitpick

**Review by jsantiago-eProsima — APPROVED** [review]

**Review by jsantiago-eProsima — DISMISSED** [review]

**Comment by MRicoIE2C** [comment]
I have seen that it lacks explanation on this chapter: https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/common.html#qos regarding ownership. 
I am going to add it now.

**Review by EduPonz — APPROVED** [review]

**Pull request merged**